### ISSUE TITLE
Does BatchNormSync support torch.nn.parallel.DistributedDataParallel?

### ISSUE THREAD

**Issue by acgtyrant** [fyu/drn#23]

**Comment by fyu** [comment]
Not really. It only supports single machine now.
